packages/expo-updates/ios/Tests/EXUpdatesLoggerOCTests.m

```diff
@@ -0,0 +1,84 @@
+//  Copyright (c) 2022 650 Industries, Inc. All rights reserved.
+
+// This test works locally, but causes compilation to fail
+// when built in the iOS unit test CI job, so commented out
+// for now.
+//
+// TODO: fix CI so this test can run
+
+/*
+#import <XCTest/XCTest.h>
+
+@interface EXUpdatesLoggerOCTests : XCTestCase
+
+@end
+
+// Headers for the Swift classes and enum
+// In application code, EXUpdates-Swift.h can be imported instead
```

Comment:
I wonder why this isn't visible to tests. It's been a few months since I did swift stuff so I don't remember if I ever got this bridge working.

That being said, I think just having the swift test is sufficient. We can rely upon the compiler to tell us if the bridge is broken.

Reply:
Replaced the OC test with a Swift test to verify the dictionary reader method